tests/Functional/DomainCommandsTest.php

```diff
@@ -29,35 +29,43 @@ class DomainCommandsTest extends TestCase
      */
     public function testDomainAddListLookupRemove()
     {
-        $domainList = $this->terminusJsonResponse(sprintf('domain:list %s', $this->getSiteEnv()));
+        // Running tests against the "test" environment instead of a multidev one since "domain:lookup" command does not
+        // search domains across multidev environments.
+        $siteEnv = sprintf('%s.test', $this->getSiteName());
```

Comment:
The repo does still contain a `TERMINUS_ENV` secret.  Would we want to use that, or a similar variable/constant, rather than hard-coding the test environment?
I'm pretty sure it's currently set to `dev` but that could be easily changed.

Reply:
In my opinion, it is easier to have a literal ("test") in place in this particular situation rather than rely on env var (it won't bring any profit).